src/components/Shared/Filters/FilterPanel.tsx

```diff
@@ -155,6 +154,14 @@ export const FilterPanel: React.FC<FilterPanelProps & BoxProps> = ({
     FinancialAccountContext,
   ) as FinancialAccountType;
 
+  const matchFilterContactStatuses = (status: string | null | undefined) => {
+    return (
+      Object.values(ContactFilterStatusEnum).find((value) => {
+        return String(value.toLowerCase()) === String(status?.toLowerCase());
+      }) || null
```

Comment:
Actually, since `value` is always uppercase, it might make more sense to convert `status` to uppercase and then compare them.

```suggestion
      Object.values(ContactFilterStatusEnum).find((value) => value === status?.toUpperCase()) || null
```